pypeit/pypeit.py

```diff
@@ -1029,14 +1029,20 @@ def extract_one(self, frames, det, sciImg, bkg_redux_sciimg, objFind, initial_sk
 
         if not self.par['reduce']['extraction']['skip_extraction']:
             msgs.info(f"Extraction begins for {self.basename} on det={det}")
+            # set the flatimg, if it exists
+            try:
+                flatimg = self.caliBrate.flatimages.pixelflat_model
```

Comment:
Is there a specific case that you have where this faults?  I'm confused because `pixelflat_model` is a component of the datamodel for `pypeit.flatfield.FlatImages`, so I would have thought that the attribute should always exist.

Reply:
It fails when a user chooses not to flatfield the data (this showed up as one of the failed dev suite tests).

Reply:
Got it.  Is there a more direct check?  Something like ``if self.caliBrate.flatimages is None``?

Reply:
Hmm... good point - I'm not sure, but I'll try the direct check and run the dev-suite to see if it breaks  😃